### DISCUSSION TITLE
Enable yay like colorful output

### DISCUSSION THREAD

**Discussion by IamRezaMousavi** [Morganamilo/paru#1194] [Q&A]
How to enable yay like colorful output?

![yay-upgrade-color](https://github.com/Morganamilo/paru/assets/83984878/9b99bcdd-a9e5-4312-a8da-d04252cc2a29)

**Comment by valsaven** [comment]


**Reply by IamRezaMousavi** [reply]
This is my pacman.conf:
```
...
# Misc options
#UseSyslog
Color
#NoProgressBar
CheckSpace
VerbosePkgLists
ParallelDownloads = 5
...
```

**Reply by yvvki** [reply]
The pacman `Color` option don't colorize the package source nor the version change.

**Reply by HubKing** [reply]
I did that, but it didn't make the versions colourful.

Paru

![image](https://github.com/user-attachments/assets/4250d5e5-ab10-47d7-8fc0-ebb7e24b8750)

Yay

![image](https://github.com/user-attachments/assets/4db9abcd-78a8-4f61-ba7e-05dbc7f9fbc5)

Isn't the colouring done by Yay itself, not by Pacman?

**Reply by aaronedev** [reply]
> I did that, but it didn't make the versions colourful.
> Isn't the colouring done by Yay itself, not by Pacman?

i think it must be? However i wonder what the option listed by paru when does: "Color: Paru only enables color if color is enabled in pacman. Enable color in your pacman.conf"

**Comment by Morganamilo** [comment]
Yay and Paru both use --upgrademenu as the flag to enable that menu. I personally don't think it adds any value and its a bit of an assault on the eyes. But it stays in Paru as a holdover from Yay. 

Personally I'd just stick with VerbosePkgLists and use --ignore if you need to ignore a pkg.

**Reply by rezad1393** [reply]
> VerbosePkgLists

I think it is actually quite a useful option.
I can see if any package change its main version (first number from left) if it follow Semantic Versioning.

**Comment by rezad1393** [comment]
> How to enable yay like colorful output?
> 
> ![yay-upgrade-color](https://private-user-images.githubusercontent.com/83984878/329278501-9b99bcdd-a9e5-4312-a8da-d04252cc2a29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg4NTYwNTEsIm5iZiI6MTczODg1NTc1MSwicGF0aCI6Ii84Mzk4NDg3OC8zMjkyNzg1MDEtOWI5OWJjZGQtYTllNS00MzEyLWE4ZGEtZDA0MjUyY2MyYTI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA2VDE1MjkxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUzMWUwZjE3NjE1MDU0ZTRiNmFjYjAzZWEwYjU4NWU3NzQ5ZWJlMWMxNDVjNThkMDY4ZWY5MDA2ZTMyNDFjNDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JCmhSuivQ_I0aaXrEVkg76g2dck8SbbU6sByjVDNCuk)

btw how old that screenshot is that nano is that version and archlinux still had community repo?

**Comment by aaronedev** [comment]
@IamRezaMousavi did you figure out how to get the colors as same as yay does? or anyone here? I think its quite useful because otherwise i just see white stuff on my screen lol

**Reply by lombervid** [reply]
> @IamRezaMousavi did you figure out how to get the colors as same as yay does? or anyone here? I think its quite useful because otherwise i just see white stuff on my screen lol

As far as I can tell, it only works using the `--upgrademenu` (or `UpgradeMenu` in the config) option. And if you want to color also the pacman packages, you need to use the `--combinedupgrade` (of `CombinedUpgrade` in the config).
